Overview.bs

```diff
@@ -349,21 +340,23 @@ Some example inputs and the corresponding expansions:
 Extensions to the Font Format {#font-format-extensions}
 =======================================================
 
-<!-- TODO: Note that these new tables are extensions specific to this specification and not part of the broader open type specifications. -->
-
-An <dfn dfn>incremental font</dfn> follows the existing [[open-type|OpenType]] format, but include two new
+An <dfn dfn>incremental font</dfn> follows the existing [[open-type|OpenType]] format, but includes two new
 [[open-type/otff#table-directory|tables]] identified by the 4-byte tags 'IFT ' and 'IFTX'. These new tables are both
 [[#patch-map|patch maps]], which encode a collection of mappings from [=font subset definition|font subset definitions=] to URIs which
 host [[#font-patch-formats|patches]] that extend the incremental font. All incremental fonts must contain the 'IFT ' table.
 The 'IFTX' table is optional. When both tables are present, the mapping of the font as a whole is the union of the mappings of
-the two tables.
+the two tables. The two new tables are used only in this specification and are not being added to the [[open-type|Open-Type]]
+specification.
 
 Note: allowing the mapping to be split between two distinct tables allows an incremental font to more easily make use of multiple
 patch types. For example all patches of one type can be specified in the 'IFT ' table, and all patches of a second type in the
 'IFTX' table. Those patches can make updates only to one of the mapping tables and avoid making conflicting updates.
 
+Note: [=incremental font|incremental fonts=] can be converted to [[WOFF2]] and still be utilized as an incremental fonts. In this
```

Comment:
It's my current understanding that the woff2 glyf transformation is predictable (while it will change the original glyf table, it will always produce the same result when decoded regardless of decoder implementation), so the encoder should be able to generate patches against the transformed and then decoded glyf table. Which is what my current prototype encoder does.

However, I'll need to spend some time reviewing the woff2 spec to make sure that is indeed a correct assumption. If I find that isn't true then I'll update the guidance here to recommend not using the glyf/hmtx transforms in woff2. Also I think I'll add a recommendation to only use per-table and/or glyph keyed patches  so that you don't have to rely on the woff2 decoders table directory and table ordering.

Reply:
If you verify that I think the section will need to be more specific, and say something about either WOFF2-encoding and then decoding the subset font prior to generating the patch or otherwise isolating how the format will change.

Also, just as we are with this spec we need to be careful about what the WOFF2 spec mandates versus suggests. Supposing the glyf transformation is predictable is that a reliable property of the spec or just of the reference implementation? If it's the latter we would need to say something about knowing how your specific encoder works (or maybe in that case it would just be better to punt).

Reply:
Agreed, on both. I'll update here once I've had a chance to review the woff2 spec in more detail.

Reply:
After a review of the woff2 spec here's my conclusions:
1. The woff2 decoder does not gaurantee a specific table ordering in the decoded file.
2. For transformed glyf/loca the woff2 decoder does not gaurantee a predictable binary encoding of the reconstructed glyf table. The transformed glyf table decodes to a list of x/y points which the decoder is free to choose how to represent in the glyf format (eg. choosing encoded widths for the coordinate values).
3. Transformed hmtx tables should have predictable decoded results.

Based on these here's what I'm proposing as guidance we give, when using IFT + a WOFF2 encoded font:
1. The IFT encoding should not use "Brotli Patches". These cannot be safely applied to an unknown table ordering resulting from the woff2 decoding.
2. The IFT encoding should not use "Per Table Brotli Patches" that modify glyf/loca IF the WOFF2 encoded contains a transformed glyf/loca table. The decoded glyf/loca is not predictable and cannot be safefy patched via a brotli patch. Glyph-keyed patches may be safely combined with a transformed glyf/loca table.
3. The IFT encoding can use "Per Table Brotli Patches" that modify transformed hmtx tables, however, it's recommended they do not out of an abundance of caution.
4. All IFT decoding/processing must happen after woff2 decoding is applied.

Reply:
Just added a new section which captures this guidance.

Reply:
Going to mark as resolved, feel free to reopen if you have additional comments about the WOFF2 guidance text.